src/main/resources/templates/ss.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+    <title>NCL</title>
+
+
+</head>
+<body>
+<table width="100%">
+    <tr>
+        <td>
+            <table align="center" width="616" auto;>
+                <tr>
+                    <td style="min-width:616px;">&nbsp;</td>
+                </tr>
+                <tr>
+                    <p>&nbsp;</p>
+                    <td style="padding:11px 35px 24px; border-top:2px solid #1eafbb;">
+                        <table>
+                            <tr>
+                                <td style="padding:0 0 3px; font:bold 36px/40px Arial;"><p>Welcome To NCL Testbed !</td>
+                            </tr>
+                            <tr>
+                                <td style="padding:0 0 13px; font:16px/24px Arial;">Hi <a href="#"  style="text-decoration:none; color:#333; font-weight:bold;">XXX</a>,</td>
+                            </tr>
+                            <tr>
+                                <td style="padding:0 0 10px; font:16px/24px Arial;">Your account approval request has been<a href="#"  style="text-decoration:none; color:#333; font-weight:bold;"> APPROVED.</a><p>Your username is: <a href="#"  style="text-decoration:none; color:#333; font-weight:bold;">xxxxx@nus.edu.sg</a></p> </td>
+                            </tr>
+                            <tr>
+                                <td style="padding:0 0 11px; font:16px/24px Arial;"><p>Should you have any question, please don't hestitate to contact us. Thanks!<br /></p>
+                                </td>
+                            </tr>
+                            <tr>
+                                <td style="padding:0 0 18px; font:16px/24px Arial; color:#333;"><p>Best Regards,<br /> The NCLTeam <br /> Tel: (65) 6601 1056 <br/><br />
+                                </p>
+                                </td>
+                            </tr>
+                        </table>
+                    </td>
+                </tr>
+                <tr>
+                    <td bgcolor="#eeeeee" style="padding:20px 10px 20px 30px;">
+                        <table width="100%">
+                            <tr>
+                                <td>
+                                    <table width="138" align="left";>
+                                        <tr>
+                                            <td style="padding:8px 0 14px; font:16px/18px Arial, Helvetica, sans-serif; color:#999;">Connect with us</td>
+                                        </tr>
+                                    </table>
+                                    <table width="426" align="right">
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this deprecated "width" attribute. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3AS1827)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this deprecated "align" attribute. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3AS1827)